.github/workflows/main.tf

```
@@ -32,7 +32,7 @@ resource "azurerm_container_registry" "musicfestival" {
 }
 
 resource "azurerm_storage_account" "musicfestival" {
-  name                     = "backendstorage"
+  name                     = "mfbackendstorage"
```

let's use explicit naming
```suggestion
  name                     = "musicfestivalsa"
```